src/v_draw.cpp

```diff
@@ -958,6 +958,20 @@ void DCanvas::DrawLine(int x0, int y0, int x1, int y1, int palColor, uint32_t re
 #endif
 }
 
+DEFINE_ACTION_FUNCTION(_Screen, DrawLine)
+{
+	PARAM_PROLOGUE;
+	PARAM_INT(x0);
+	PARAM_INT(y0);
+	PARAM_INT(x1);
+	PARAM_INT(y1);
+	PARAM_INT(color);
```

Comment:
@coelckers tagging you so you see the above question, since you haven't responded